ipfs-6-to-7/gx/ipfs/QmNtxDQBrVzy88FEjVikyM5tAbyfnHm5jJbJZcUhVykusq/snappy/encode_amd64.go

```diff
@@ -26,4 +26,4 @@ func extendMatch(src []byte, i, j int) int
 // encodeBlock has the same semantics as in encode_other.go.
 //
 //go:noescape
-func encodeBlock(dst, src []byte) (d int)
\ No newline at end of file
+func encodeBlock(dst, src []byte) (d int)
```

Comment:
Ah, these are vendored deps. We should just leave them alone.

Reply:
so just remove the `go fmt` commit? Can do, although it means `go fmt ./...` is always going to cause problems in this repo

Reply:
Done

Reply:
Yeah. Unfortunately, our vendoring game here could use some work.